src/Features/Core/Portable/CodeRefactorings/SyncNamespace/AbstractChangeNamespaceService.cs

```diff
@@ -744,7 +744,7 @@ await FixReferencesAsync(document, changeNamespaceService, addImportService, ref
 
             var fixedDocument = editor.GetChangedDocument();
             root = await fixedDocument.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            var result = (fixedDocument, containers.SelectAsArray(c => root.GetCurrentNode(c)));
+            var result = (fixedDocument, containers.SelectAsArray(c => root.GetCurrentNode(c)!));
```

Comment:
Why the ! here? I admit it's not clear why this one is safe.

Reply:
What's the suggested solution here?

Reply:
I guess rather than just using a !, do some loop and throw if there's a null. Because unless the potential null is being handled elsewhere, this isn't something we want escaping from this function. Specifically, if we throw here then it means some of our crash reporting can report the crash with the document editor or state and we might be able to sort out what happened. If we just return null and something higher up crashes, it's much harder to debug post-mortem.